src/main/kotlin/uk/gov/justice/digital/hmpps/sentenceplan/controller/PlanController.kt

```diff
@@ -32,8 +42,18 @@ class PlanController(
   }
 
   @GetMapping("/{planUuid}")
+  @Operation(
```

Comment:
Should this not document more `@ApiResponse` values, as it can return non-success if an unknown uuid is submitted?

Reply:
I didn't bother doing GET because we've already implemented it and I didn't want to start adding API responses without looking deeper into how the controller already responds.

Reply:
I've added some now but no guarantees they actually match the expected implementation
https://github.com/ministryofjustice/hmpps-sentence-plan/pull/90/commits/3f199955bd0825bd26daad8eb661b5fdfc429e0a